cli/command_snapshot_create.go

```diff
@@ -290,7 +290,16 @@ func getLocalBackupPaths(ctx context.Context, rep repo.Repository) ([]string, er
 	var result []string
 
 	for _, src := range sources {
-		if src.Host == rep.ClientOptions().Hostname && src.UserName == rep.ClientOptions().Username {
+		// add all sources belonging to the repository user@host
+		// ignore sources that have Manual field set to true in the SchedulingPolicy
+		policyTree, err := policy.TreeForSource(ctx, rep, src)
+		if err != nil {
+			return nil, errors.Wrapf(err, "unable to get policy tree for source %v", src)
+		}
+
+		if src.Host == rep.ClientOptions().Hostname &&
```

Comment:
perhaps make a helper here - the same helper will be used in source manager.

Reply:
I was testing source manager and realized that it's already skipping sources where the `nextSnapshotTime` is not set. Since we remove `interval` and `timesOfDay` fields from the scheduling policy, the `refreshStatus` will not set any `nextSnapshotTime` for these sources.
Where would the explicit handling happen if required?